docs/conceptual/copy-and-update-record-expression-[fsharp].md

```diff
@@ -0,0 +1,40 @@
+---
+title: Copy and Update Record Expression (F#)
```

Comment:
Same as the other - `expressions`. Also probably worth updating the file name :+1: